pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -735,7 +742,11 @@ pub mod pallet {
 				);
 
 				if affiliate_config.mode == AffiliateMode::Open && affiliate_config.enabled_in_buy {
```

Comment:
If we think about this from the sage context, I think it is the fee handler itself, which should determine if it is enabled or not, but maybe we do it in a second step. It is not obvious to me now, how we best implement this, as all the config stuff is in the AAA. What do you think, could we move this config stuff into the affiliates-pallet?

Reply:
I think the best way to make this independent is to, much like we did for other components, define a trait or set of traits that allow the inspection of the configuration, that way the sage-pallet only needs to implement the traits and become the configuration provider, or we could even define a separate pallet just to hold the configuration if we find that better.

TLDR: Yes, this should be discussed in the second round of development

Reply:
Ok, cool, I agree. I generally wonder, if global config needs to be defined aggregated in the core pallet or if it would be much better to configure the pallet sovereignly in this case too.

Reply:
I think that all config needed to support the features the core pallet has should be there, but we need to think on which config has to be shared and how customizable that configuration can be in terms of types and generics.

Reply:
Exactly, I agree with what you say, but we should revisit, which things are actually also needed by the core pallet itself.

Reply:
but I do think, again, that with the right abstractions we can move some stuff out of the core pallet.